jsapp/js/account/stripe.types.ts

```diff
@@ -147,6 +147,8 @@ export interface Organization {
   modified: string;
   slug: string;
   is_owner: boolean;
+  is_mmo: boolean;
```

Comment:
Should these be optional until the backend will actually return it? Or just returning false in the query is safe enough

Reply:
As far as I know, and from my tests, backend is already returning these fields. I'm setting it to false in the current query because when talking to @jamesrkiger we thought it would be a good idea to force the default "false" values until we have the front end properly implemented to avoid issues.